src/style.css

```diff
@@ -1,3 +1,84 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Ensure that the font 'Roboto' is correctly loaded in your HTML file. This is important for the font-family declaration to work as expected.